sdk/core/Microsoft.Extensions.Azure/samples/Startup.cs

```diff
@@ -52,7 +52,7 @@ public void ConfigureServices(IServiceCollection services)
                 builder.ConfigureDefaults((options, provider) =>  options.AddPolicy(provider.GetService<DependencyInjectionEnabledPolicy>(), HttpPipelinePosition.PerCall));
 
                 builder.AddBlobServiceClient(Configuration.GetSection("Storage"))
-                        .WithVersion(BlobClientOptions.ServiceVersion.V2018_11_09);
+                        .WithVersion(BlobClientOptions.ServiceVersion.V2019_02_02);
```

Comment:
Should this necessarily be changed here? @tg-msft ?

Reply:
Yes, there was a PR that changed the version and broke the build.